readme.md

```diff
@@ -62,16 +68,16 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
-$ touch README.md
-$ git push origin master
-$ git push origin master
-$ git remote add origin git@github.com/username/project-repo.git
-$ git remote add juan git@github.com/juan/project-repo.git
-$ git commit -m "initial commit"
 $ cd project-repo
 $ git init project-repo
```

Comment:
I didn't even think that was an unfamiliar command, I guess usually we just used "git init" inside a new repo and I didn't even think about the fact that this one actually used the repo name in the command. Learned something new here!